dev/format_python_code.sh

```diff
@@ -15,26 +15,5 @@ if ! ${ISORT} --version >/dev/null 2>&1; then
     exit 1
 fi
 
-# The commands must be run on each module directory separately,
-# otherwise tools confuse the "current" module
-for paths in \
-    "cvat-sdk" \
-    "cvat-cli" \
-    "tests/python/" \
-    "cvat/apps/quality_control" \
-    "cvat/apps/analytics_report" \
-    "cvat/apps/engine/lazy_list.py" \
-    "cvat/apps/engine/background.py" \
-    "cvat/apps/engine/frame_provider.py" \
-    "cvat/apps/engine/cache.py" \
-    "cvat/apps/engine/default_settings.py" \
-    "cvat/apps/engine/field_validation.py" \
-    "cvat/apps/engine/model_utils.py" \
-    "cvat/apps/engine/task_validation.py" \
-    "cvat/apps/dataset_manager/tests/test_annotation.py" \
-    "cvat/apps/dataset_manager/tests/utils.py" \
-    "cvat/apps/events/signals.py" \
-    ; do
-    ${BLACK} -- ${paths}
-    ${ISORT} -- ${paths}
-done
+${BLACK} .
+${ISORT} --resolve-all-configs .
```

Comment:
I'm a little bit worried that these commands now don't have any reference to CVAT specifically, so starting them from a directory outside of CVAT repo may have unexpected consequences. This doesn't look like a big problem, but maybe add a check that the current directory is a path inside a CVAT repo?

Reply:
I changed the script so that it would always format the CVAT repo.